frontend/src/components/account/ChangePassword/ChangePassword.module.scss

```diff
@@ -0,0 +1,5 @@
+.title {
+  font-size: 20px;
+  font-weight: 400;
```

Comment:
It is a default font-weight

Reply:
This style for h2, therefore I need font-weight